tgui/packages/tgui-panel/styles/goon/chat-dark.scss

```diff
@@ -490,7 +490,7 @@ em						{font-style: normal;	font-weight: bold;}
 .headofpersonnel {color: #526bf8}
 .headofsecurity {color: #ec3550}
 .janitor {color: #81dacb}
-.lawyer {color: #c07d7d}
+.corporateaffairsagent {color: #50c878}
```

Comment:
this should also be set in the stylesheet

Reply:
not sure what you mean by this, if it's stylesheet.dm then that doesn't have any job specific stuff, that's for different things. this is where you set the chat color for specific jobs.